### PR TITLE
{tools}[fosscuda/2018b] Meson v0.48.1 (w/ Python 3.6.6) + Ninja v1.8.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/Meson/Meson-0.48.1-fosscuda-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ef893e2d64083463fe41f05d683edbe8c44bc187cd9cc66cbc2b1ce399567447']
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('Ninja', '1.8.2'),
+]
+
```
Comment (boegel) [inline easybuild/easyconfigs/m/Meson/Meson-0.48.1-fosscuda-2018b-Python-3.6.6.eb]: @akesandgren Missing `download_dep_fail = True` and `use_pip = True`

Comment (akesandgren) [reply easybuild/easyconfigs/m/Meson/Meson-0.48.1-fosscuda-2018b-Python-3.6.6.eb]: I blame the foss-2018b version in develop...

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/88af60cf53e7eca61bcfc3a935735b03 for a full test report.

**Comment by akesandgren** [comment]
@vanzod @boegel ping on mergeing this?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar04.gastly.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/be09226ae27154df836276935fc641d1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2438.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/695b3f545c463742aeb1514464759d42 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2049.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8709dfa882e71c55bdcd30a31bedfa44 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**